src/cloudinary/index.js

```diff
@@ -0,0 +1,23 @@
+
+import { Cloudinary, Util, Configuration } from 'cloudinary-core';
```

Comment:
What is this file for?

Reply:
this is wrapper for Cloudinary API, so that we don't face the problem in the future when we need to change api, and to make it more readable instead of using `url` and `video_url` directly from the SDK.